src/components/Events/EventCard.tsx

```diff
@@ -25,7 +25,7 @@ export const EventCard: React.FC<EventCardProps> = ({
     setIsLiked(!isLiked);
   };
 
-  const getImagePath = (imageId: number) => `/${(imageId % 12) + 1}.png`;
+  const getImagePath = (eventPrice: number) => `/${(eventPrice % 12) + 1}.png`;
```

Comment:
Is the image based on price? I would expect sport or something 😂

Reply:
This is for now just randomized images - I will have a PR with a folder/images for each sport, but currently, I don't have a list of available sports. I need the list from either the backend, or we need to create a list of, for example, 10 sports for now and scale it later. Before the bug, it was still random based on the id number. Before, it worked with the ID, but later you changed it to something like vx5hu4578sc67... soooo 😅